jenkins/scripts/integration_test.sh

```diff
@@ -49,9 +49,9 @@ fi
 
 if [ "${IMAGE_OS}" == "Ubuntu" ]
 then
-  TEST_EXECUTER_FLAVOR="${TEST_EXECUTER_FLAVOR:-4C-16GB-200GB}"
+  TEST_EXECUTER_FLAVOR="${TEST_EXECUTER_FLAVOR:-4C-8GB-200GB}"
```

Comment:
I'm wondering if we can reduce it to 16GB in line 47 as well

Reply:
I didn't want to touch something that I have not been working with. In theory, I would say yes, but I'll leave it up to you.

Reply:
Yes, sure. I'll check that.